README.md

```diff
@@ -10,115 +10,62 @@ This repo is meant as a template which you can use to create a Discord bot very
-    // The user ids for the support team
-    botSupporters: [] as string[],
-    // The user ids for the other devs on your team
-    botDevs: [] as string[],
-    // The user ids who have complete 100% access to your bot
-    botOwners: [] as string[],
-  },
-};
-```
-
-4. Start the bot `deno run -A --quiet mod.ts`
-
-**Note:** To run the bot with [PM2](https://github.com/Unitech/pm2): `pm2 start mod.ts --interpreter="deno" --interpreter-args="run -A --quiet -r" `
-
-## Step By Step with Docker
-
-You can also run this image with Docker.
+## Step By Step with Docker Compose
 
-1. Install `docker` [Getting started with docker](https://docs.docker.com/get-started/)
-2. Clone the repository
-3. Create your `configs.ts` 
-4. Build the container, from the directory containing the repository `docker build -t mybot .`
-5. Run the container with `docker run -v $(pwd)/configs.ts:/bot/configs.ts -t mybot`
 
+1. Clone the repository
+2. Get your discord bot's token (not the application id or key) from your [Discord Developer console](https://discord.com/developers/applications).
+3. Create an `.env` file in your project's root directory and add your bot's token with the command `echo TOKEN=123123123 > .env`
+4. Install Docker Compose if you haven't already (check with `docker-compose -v`) [https://docs.docker.com/compose/install/](https://docs.docker.com/compose/install/)
+5. start your container with `docker-compose up -d` 
 
-**Notes:** 
-- In the previous commands `$(pwd)` and `.` can be replaced with the full path to directory
-- You can also use the `-d` argument to run the container in background
+### How to use docker-compose
+   - The flag `-d` runs the container in the background in "Detached mode". If you want to see the console output you can remove the flag or inspect your container using `docker-compose logs mybot`. 
+   - The `--build` flag forces a build. If this is left out, and nothing is
+   - You can also access your docker container's shell with the command `docker-compose exec mybot sh`  Use `ctrl - d` to quit the session, your container will still run.
+   - Inspect your `docker-compose.yml` if you want. This file tells `docker-compose` to load your `.env` and where your `Dockerfile` is located. `docker-compose` uses that buildfile to get the latest version of `deno`, set up the environment, copy files from your local project folder, set permissions and finally start the bot in the docker container.
+   - For more information, visit [https://docs.docker.com/compose/reference](https://docs.docker.com/compose/reference)
 
-### With docker-compose
 
-1. Install `docker`
-2. Build the container following the steps (up to the fourth) mentionned above 
-3. Install `docker-compose`. [Getting started with docker-compose](https://docs.docker.com/compose/gettingstarted/)
-4. Create a `docker-compose.yml` with this content
+## Step By Step using Docker (without Docker Compose)
```

Comment:
I don't think this is needed here either. I would keep this readme minimal and link towards guides